akka-stream-tests/src/test/scala/akka/stream/io/TlsSpec.scala

```diff
@@ -377,8 +377,8 @@ class TlsSpec extends StreamSpec(TlsSpec.configOverrides) with WithLogCapturing
                   override def onPush() = push(out, grab(in))
                   override def onPull() = pull(in)
 
-                  override def onDownstreamFinish() = {
-                    system.log.debug("me cancelled")
+                  override def onDownstreamFinish(cause: Throwable) = {
+                    system.log.debug(s"me cancelled, cause {}: {}", cause.getClass.getName, cause.getMessage)
```

Comment:
I'd write that as
```
debug("me cancelled, cause {}", cause)
```

since toString of exceptions typically include the class name

but it's just a test so doesn't matter